### ISSUE TITLE
AppPagination value transformation

### ISSUE THREAD

**Issue by gsiuniak** [EthVM/ethvm#353]
Currently, API calls use zero-based system for pagination calls. I.E. a human readable "Page 1" requires calling for page 0 via the API.

AppPaginate.vue should be updated to accept a zero-based number in, display a human-readable string, and on update, return a zero-based number to the parent component.

**TLDR**: AppPaginate should input and output zero-based page numbers from and to its parent component.

**Comment by gsiuniak** [comment]
@aldoborrero 

I update AppPaginate.vue to do this: https://github.com/EthVM/ethvm/tree/bug/PageTx-pagination
However, views need to be updated to utilize the method. 

It shouldn't be too hard, because views/components are/were previously juggling the transformation of a zero-based "page" value and a human-readable "page" value. They have to be normalized to zero-based.

Also, can you take a look at PageTxs.vue? I put a comment:
// Aldo: look here //

When implementing this view, I copied code from before which included:
```
this.from = page > this.page ? this.txs[this.txs.length - 1].getBlockNumber() : this.txs[0].getBlockNumber()
```

I'm not sure how it is supposed to work properly. I get the idea of a starting "from" a block number, but it doesn't quite work correctly. The last item in the table for page 1 will be the first item in page 2. (This makes some sense, as we are starting "from" the last block from our first result array). But to go back a page...

The same "from" block number is used as the previous call, so we can't go back. I tried subtracting MAX_ITEMS from the block number, and that gives some odd results. Not sure if the blocks are linearly numbered? Tried sorting differently....

If you can take a look at that.

**Comment by aldoborrero** [comment]
thanks @gsiuniak I've already merged this branch into `feature/pagination-improvements` branch. So I'm continuing from there.

I also detected why for blocks and uncles was not behaving as properly:

- The data-set you were using had a flaw (is already corrected in develop). So make sure you do a docker-compose down -v and later the usual stuff.

The code for the txs pagination were not very well designed neither on API and front-end (so I'm taking a look at that).

Continue with remaining parts on attention-to-detail card
